packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -4420,7 +4420,7 @@ YoEdncuy+GQGzE9yLOhC4HNfHQXpqp2tMPdRlw==</ds:X509Certificate>
 
 <ApiOperation method="post" url="/api/v1/apps/${applicationId}/credentials/csrs" />
 
-Generates a new key pair and returns the Certificate Signing Request for it
+Generates a new key pair and returns the CSR (Certificate Signing Request) for it. The information in a CSR is used to verify and create your SSL certificate. It also contains the public key that will be included in your certificate.
```

Comment:
Reference: https://info.ssl.com/what-is-a-csr/

Reply:
Thanks for the reference. I would double-check that this is standard and Okta doesn't have its own rules in the process.

Reply:
Will check with `Federations` team as per the ownership info at https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/650676459/Developer+Doc+Ownership.

Reply:
Okta does not have special rule about it.
Nit: the certificate here is not used by SSL and also it might call out it is used by CA:
The information in a CSR is used by the Certificate Authority (CA) to verify and create your ~SSL~ certificate

Reply:
@yuliu-okta thanks! Addressed your comment.